internal/k8s/crdcontroller.go

```diff
@@ -698,7 +698,7 @@ func validateHostRuleObj(key string, hostrule *akov1alpha1.HostRule) error {
 		}
 	}
 
-	if hostrule.Spec.VirtualHost.TCPSettings != nil {
+	if hostrule.Spec.VirtualHost.TCPSettings != nil && len(hostrule.Spec.VirtualHost.TCPSettings.Listeners) > 0 {
```

Comment:
Do we have to inform the user in such a scenario?

Reply:
@swathinsankaran `SSL enabled ` error should be raised only when listeners are specified. TCPSetting can have other setting without specifying listeners e.g. laodBalancerIP.

Reply:
So we do not require to notify anything to customer